inputs_land.md

```diff
@@ -17,27 +17,32 @@ Table 1: External inputs used by the land model <sup>[1](#table_footnote)</sup>
 | Name | Description | Type | Source | Resolution | Unit |
 | :--- | :--- | :--- | :--- | :--- | :--- |
 | Historical Land Use and Land Cover | Land area by region, land type and year. Land cover data is provided beginning in 1700 in order to spin-up the carbon cycle within GCAM. Crop-specific harvested area is used to downscale FAO data to a subnational level; however this data is only available for a single year. Similarly, the division between irrigated and rainfed land is only available for a single year only. | External Data Set | <a href="https://github.com/JGCRI/moirai">Moirai</a> | Raw data is by country, water basin and land type. | thousand $$km^2$$ |
-| Historical Harvested Area | Harvested land area by country, crop, and year. | External Data Set | FAO | Raw data is by country, crop, and year | Hectares |
+| Historical Harvested Area | Harvested land area by country, crop, and year. | External Data Set | FAO, gcamdata-faostat | Raw data is by country, crop, and year | Hectares |
```

Comment:
`gcamdata-faostat` is the name of your pre-processing package?  Is it publicly available yet?

Reply:
Not yet. Working on it now. Hopefully, we will have a version that can share per request soon.

Reply:
Links updated. Thanks!